docs/css/react.scss

```diff
@@ -122,6 +122,7 @@ h1, h2, h3, h4, h5, h6 {
 .hash-link {
   color: $mediumTextColor;
   display: none;
+  padding-left: 8px;
```

Comment:
It's compensating... changing how we add space between the title and hash-link. The UI should look the same:

![hello_world_-_react_and_hello_world_-_react](https://cloud.githubusercontent.com/assets/1256329/22855571/b5be3e10-f050-11e6-8d59-63d651e98abe.png)